ontology/docs/faq.md

```diff
@@ -0,0 +1,115 @@
+
+## What device data should be modeled?
+**Model what you need for the applications and analytics that are reasonably anticipated.** 
+
+More data can always be added later (as long as it’s exposed in the gateway). The general rule is to model things that describe the device’s general behavior without superfluous detail regarding its low-level configuration; however, anything can be modeled, so if need arises then the ontology can be extended to accommodate. There are a few types of data that devices typically sends:
+
+### Measured Telemetry
+These are sensors associated with the device, such as `supply_air_temperature_sensor` and `chilled_water_valve_percentage_sensor`. They are directly measured (or calculated) by the device and return updated values as the state of the device changes through time. All measured telemetry is normally modeled. *NOTE: we do not consider calculated telemetry differently from raw telemetry.*
+
+### Setpoint Telemetry
+These are values to which measured telemetry values are being directly controlled, such as `zone_air_cooling_temperature_setpoint` and `supply_air_static_pressure_setpoint`. The measured values are compared to the setpoint values to provide some type of state comparison (e.g. the zone is cold), whereby the device can respond through adjustment of control values. All setpoints are normally modeled. 
+
+### Control States
+These are values that represent how the device is responding to measured deviations from setpoints or other changes in environmental/temporal conditions. Such fields as `supply_fan_run_command` and `chilled_water_valve_percentage_command` are examples of control states that the device may adjust directly or indirectly based on measured conditions (e.g. the time of day for scheduled operation of the fan, or setpoint deviation for position of the valve). Most control states are normally modeled.
+
+### Interpreted States
+These are states which the device interprets from underlying data (whether it is exposed or not). The best example of this is an alarm: for example, a fan mismatch alarm compares the status of the fan to the command and returns an alarm if they are no equal. Interpreted states are not normally modeled.
+
+### Configuration Information
+These are internal device points which are used to configure the way in which the device attempts to maintain control values, or how its components control internally. Stage up/down timers, PID gains, etc.; configuration information is rarely modeled. 
+
+## How do you extend the ontology?
```

Comment:
can we mentioned that the ontology type validator is triggered upon a pull request.
Keep iterating until the Ontology Type validator passes which means that the content is valid?

Reply:
added